components/HorizontalScroller.js

```diff
@@ -1,68 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ChevronLeft } from '@styled-icons/fa-solid/ChevronLeft';
-import { ChevronRight } from '@styled-icons/fa-solid/ChevronRight';
-import themeGet from '@styled-system/theme-get';
+import { ArrowBack } from '@styled-icons/material/ArrowBack';
+import { ArrowForward } from '@styled-icons/material/ArrowForward';
 import styled from 'styled-components';
 
 import { debounceScroll } from '../lib/ui-utils';
 import withViewport from '../lib/withViewport';
 
-/** Main chevrons container, spacing the items properly */
-const ChevronsContainer = styled(({ onPrevClick, onNextClick, ...props }) => (
-  <div {...props}>
-    <ChevronLeft onMouseDown={onPrevClick} disabled={!onPrevClick} />
-    <ChevronRight onMouseDown={onNextClick} disabled={!onNextClick} />
-  </div>
-))`
+import { Box, Flex } from './Grid';
+
+const RefContainer = styled(Box)`
   display: flex;
-  justify-content: space-between;
-  color: #dadada;
-  user-select: none;
-  width: ${props => props.size * 3.5}px;
+  overflow-x: auto;
+  scroll-behavior: smooth;
+`;
 
-  svg {
-    width: ${props => props.size}px;
-    height: ${props => props.size}px;
+const ControlsContainer = styled(Flex)`
+  z-index: 10;
+  position: relative;
+  top: 30rem;
+`;
 
-    &:not([disabled]) {
-      color: ${themeGet('colors.primary.500')};
+const ArrowContainer = styled.button`
```

Comment:
Can we rather use a `StyledRoundButton`? We want to have the default button styles & docus/hover/active states for this, it's simpler to re-use existing components for that.